lib/client/jsx/api/magma_api.js

```diff
@@ -1,57 +1,56 @@
 import {
-  parseJSON, generateDownload, makeBlob, checkStatus, headers
+  parseJSON,
+  generateDownload,
+  makeBlob,
+  checkStatus,
+  headers
 } from '../utils/fetch_utils';
 
 const magmaPath = (endpoint) => `${TIMUR_CONFIG.magma_host}/${endpoint}`;
 
 const magmaPost = (endpoint, exchange, params) => {
-  return exchange.fetch(
-    magmaPath(endpoint),
-    {
+  return exchange
+    .fetch(magmaPath(endpoint), {
       method: 'POST',
       credentials: 'include',
       headers: headers('json'),
       body: JSON.stringify({
         ...params,
-        project_name: TIMUR_CONFIG.project_name,
+        project_name: TIMUR_CONFIG.project_name
       })
-    }
-  )
+    })
     .then(checkStatus)
     .then(parseJSON);
-}
+};
 
-const create = (name, attributes)=>{
+const create = (name, attributes) => {
   let element = document.createElement(name);
-  for(let key in attributes){
-    element.setAttribute(key,attributes[key]);
+  for (let key in attributes) {
+    element.setAttribute(key, attributes[key]);
   }
   return element;
 };
 
-const input = (name, value)=>{
+const input = (name, value) => {
   return create('input', {type: 'hidden', name, value});
 };
 
-export const getTSVForm = (model_name, filter)=>{
-  let { Authorization } = headers('auth');
+export const getTSVForm = (model_name, filter, attributes) => {
+  let {Authorization} = headers('auth');
   let data = {
     'X-Etna-Authorization': Authorization,
     project_name: TIMUR_CONFIG.project_name,
     model_name,
     record_names: 'all',
-    attribute_names: 'all',
+    attribute_names: attributes,
```

Comment:
Change `attributes` and pass it in as an argument.

Reply:
Could be attribute_names for greater clarity and compactness, "attributes" might be a collection of attribute objects